MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -1049,6 +1101,132 @@
          When pressed the download list is sorted in download date descending order. -->
     <string name="downloads_sort_download_size_desc">Largest</string>
 
+    <!-- This string is displayed in the middle of the addons list while the addons is being loaded. -->
+    <string name="addons_title">Add-ons</string>
+
+    <!-- This string is displayed in the title bar when a web extension session is displayed. -->
+    <string name="web_extensions_title">Web Extension</string>
+
+    <!-- This string is displayed in the middle of the addons list while the addons is being loaded. -->
+    <string name="addons_loading">Loading Add-ons</string>
+
+    <!-- This string is displayed under the addons title in the addons list. -->
+    <string name="addons_description">Add-ons Description</string>
+
+    <!-- This string is displayed in the Addon Details description text when no description is available. -->
+    <string name="addons_no_description">No description.</string>
+
+    <!-- This string is displayed in the Addon Details authors property text. -->
+    <string name="addons_authors_title">Authors</string>
+
+    <!-- This string is displayed as the Addon Details authors property text value when no authors available. -->
+    <string name="addons_no_authors">No authors.</string>
```

Comment:
There is a period here, but there isn't above. Is that expected?

Reply:
Removed.